src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -604,7 +604,7 @@ export default defineComponent({
 
   .content {
     background-color: variables.$color-bg-4;
-    padding-bottom: 120px;
+    padding-bottom: 12px;
```

Comment:
The transaction info will look like that after your changes.
![image](https://github.com/superhero-com/superhero-wallet/assets/9008074/495982de-d71c-4e5c-995a-6dae59ad5ae2)
instead of this
![image](https://github.com/superhero-com/superhero-wallet/assets/9008074/8a9d41c1-c4e2-4a87-b8f3-1b9c2b4f95e7)

Reply:
On my screen it looked like a mistake, as there was a huge gap at the bottom of the page. I will revert this as it is unrelated anyway